department-of-veterans-affairs/caseflow
# Add Assignee Label to CachedAppealAttributes for AMA Appeals

Bumps #11316 

### Description

 - Adds the Assignee Label to the CachedAppealAttributes Table
 - Populates that column for AMA appeals
 - *(Next PR - Legacy Appeals)*
 - *(Last PR - Backend Sorting)*
### Acceptance Criteria
- [x] Migration runs successfully
- [x] Job caches Assignee Label for each AMA Appeal

### Testing Plan
1. Migrate your DB
1. Run the UpdateCachedAppealAttributes job
1. Confirm all AMA appeals in the DB have an assignee_label

*Schema Changes Only*
